c/shared/source/uforead/uforead.c

```diff
@@ -1001,6 +1016,62 @@ static void setFontMatrix(ufoCtx h, abfFontMatrix* fontMatrix, int numElements)
     freeValueArray(h);
 }
 
+static void setFlexListArrayValue(ufoCtx h) {
+    int i = 0;
+    if (h->valueArray.cnt == 0)
+        return;
+    while ((i < h->valueArray.cnt)) {
+        char* pointName = memNew(h, strlen(h->valueArray.array[i]));
+        strcpy(pointName, h->valueArray.array[i]);
+        *dnaNEXT(h->hints.flexOpList) = pointName;
+        i++;
+    }
+    freeValueArray(h);
+}
+
+static void setStemsArrayValue(ufoCtx h, HintMask* curHintMask) {
+    StemHint* stem;
+    float pos = 0;
+    float width = 0;
+    int stemFlags = 0;
+    int count = 0;
+    int isH = !(stemFlags & ABF_VERT_STEM);
+//    if ((transform != NULL) && (!transform->isOffsetOnly)) { looks like transform is ALWAYS NULL
+//        /* We omit stems if the stems are being skewed */
+//        if (isH && (transform->mtx[2] != 0.0))
+//            return result;
+//        if ((!isH) && (transform->mtx[1] != 0.0))
+//            return result;
+//    }
```

Comment:
Remove WIP(?) commented-out code (or check/fix re: `transform`)

Reply:
I actually put these back just in case. Now I keep transform within the parseState struct in ufoCtx, so it's accessible anytime.